observing_examples.ipynb

```diff
@@ -0,0 +1,585 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <center>Computing the count rate of observing <a href=\"https://en.wikipedia.org/wiki/HAT-P-11\">HAT-P-11</a> with the APO 3.5m telescope using <tt>synphot</tt></center>"
```

Comment:
It'd be great if you gave some narrative motivation in a new cell at the top of the notebook. [It might look like this nb](https://github.com/astropy/astropy-tutorials/blob/master/tutorials/notebooks/synthetic-images/synthetic-images.ipynb), for example.

Reply:
You mean like an "objectives" list under the title?
(p.s. I do have a title, but if you're looking at the notebook on github it doesn't show up for some reason)

Reply:
For now, let's copy the format of the other notebook directly with authors, objectives, keywords, and an outline of what's to come in the notebook (we'll worry about making it link to the parts of the notebook later).